addons/html_editor/static/src/main/chatgpt/chatgpt_prompt_dialog.xml

```diff
@@ -14,7 +14,7 @@
                 </div>
                 <div class="w-100 o-min-width-0">
                     <div class="d-flex flex-wrap align-items-baseline mb-1 lh-1">
-                        <strong class="me-1 text-truncate"><t t-esc="message.author === 'user' ? 'You' : 'OdooBot'"/></strong>
+                        <strong class="me-1 text-truncate"><t t-esc="authorName"/></strong>
```

Comment:
Could as well use `t-if` and `t-else` here 🤷 Unless we want the OdooBot name dynamically fetched.